app/src/main/java/sdp/moneyrun/Game.java

```diff
@@ -108,6 +108,18 @@ public Game(String gameId,
         this.isVisible = isVisible;
     }
 
+    public Game(String gameId,
```

Comment:
We should maybe think to have a Builder as an upcoming task, since we might add more arguments in the upcoming weeks and there are already quite a few arguments